### ISSUE TITLE
Add a button to open code in editor

### ISSUE THREAD

**Issue by rlamana** [okteto/docker-desktop-extension#73]
**Is your feature request related to a problem? Please describe.**
When a remote environment is launched and it's ready for development it would be very useful to take users directly to the code to start developing on the new environment. 

**Describe the solution you'd like**
Based on the path to the docker-compose file, a button can be added to open an editor at the same path. On a first iteration, we can create just an "Open in VSCode" button, being VSCode one of the most popular editor nowadays. But in the future we could add support for other editors.

- VSCode button mock:
![VSCode Button Mock](https://user-images.githubusercontent.com/237819/191768386-7d7a8c28-0d50-48f9-8fc3-67da06210b48.png)

- Editor selector mock:
![VSCode Button Mock - 2](https://user-images.githubusercontent.com/237819/191768417-c49ee6cb-2166-4a64-be46-199a6faf7497.png)

**Comment by semi-infiknight** [comment]
please assign me this issue

**Comment by pchico83** [comment]
Awesome @semi-infiknight , thanks for participating in Hacktoberfest!

**Comment by pchico83** [comment]
@semi-infiknight would you be able to work on this?
This would be a great addition. Docker is offering 15 personalized macro keypads to the best docker extension contributors worldwide, and we would like to propose this issue to the contest

**Comment by semi-infiknight** [comment]
ok i will

**Comment by ckragarwal** [comment]
Can I work on this if this is still pending?

**Comment by HeartBeat1608** [comment]
Hey @rlamana, This one seems to be dormant for quite some time. Is it still actively being developed ?

**Comment by rlamana** [comment]
Not at all. Since @semi-infiknight couldn't continue with this issue, @HeartBeat1608 or @ckragarwal if you are still interested, this would be a great addition to the extension.
